govcd/edgegateway.go

```diff
@@ -1378,31 +1368,31 @@ func getVnicIndexByNetworkNameAndType(networkName, networkType string, vnics *ty
 }
 
 // getNetworkNameAndTypeByVnicIndex looks up network type and name in list of edge gateway interfaces
-func getNetworkNameAndTypeByVnicIndex(vNicIndex int, vnics *types.EdgeGatewayVnics) (string, string, error) {
+func getNetworkNameAndTypeByVnicIndex(vNicIndex int, vnics *types.EdgeGatewayInterfaces) (string, string, error) {
 	if vNicIndex < 0 {
 		return "", "", fmt.Errorf("vNic index cannot be negative")
 	}
 
 	foundCount := 0
 	var networkName, networkType string
 
-	for _, vnic := range vnics.Vnic {
+	for _, vnic := range vnics.EdgeInterface {
 		if vnic.Index != nil && *vnic.Index == vNicIndex {
 			foundCount++
-			networkName = vnic.PortgroupName
+			networkName = vnic.Name
 			networkType = vnic.Type
 		}
 
-		// Search inside "subinterface tree"
-		if vnic.Type == types.EdgeGatewayVnicTypeTrunk && len(vnic.SubInterfaces.SubInterface) > 0 {
-			for _, subInterface := range vnic.SubInterfaces.SubInterface {
-				if subInterface.Index != nil && *subInterface.Index == vNicIndex {
-					foundCount++
-					networkName = subInterface.LogicalSwitchName
-					networkType = types.EdgeGatewayVnicTypeSubinterface
-				}
-			}
-		}
+		// // Search inside "subinterface tree"
```

Comment:
Please either remove the commented block or add a comment explaining why we need it.

Reply:
Oh, sorry - leftover. Removed.